frontend/src/types/models/index.ts

```diff
@@ -0,0 +1,5 @@
+export interface BaseModel {
+  id: string;
```

Comment:
We have set this as a UUID in the backend, I don't know if you want to enforce that here as well.